src/targets/services/fetchOpenPathTrips.js

```diff
@@ -59,6 +60,8 @@ const fetchOpenPathTrips = async () => {
   client.setAppMetadata(appMetadata)
 
   await fetchTrips(client, account)
+
+  await checkAndSendGeolocationQuotaNotification(client, logService)
```

Comment:
@paultranvan what do you think about checking and sending a notification here ? Should I create a new service, and start the new service from `fetchOpenPathTrips` service ?

Reply:
Je serais pour ma part plutôt pour avoir un nouveau service.

Reply:
I think it's good here :+1:

Reply:
On s'est croisé avec @Merkur39 :D Pourquoi un nouveau service  ?
ça me semble alourdir pour pas grand chose

Reply:
Ca me semble "clean" d'avoir un service de notification, mais si pour le moment un seul service l'utilise allons-y comme ça :)

Reply:
Je suis assez d'accord avec l'approche de mettre la notif dans ce service. Que fait ce service ? 

Il récupère les trips sur OpenPath. Et vu qu'on veut réagir justement pendant cette récupération (ici envoyer une notif si y en a trop), ça me semble tout à fait souhaitable de l'avoir ici plutôt qu'ailleurs. Ça reste exactement dans le même domaine fonctionnel et à partir de ce moment là ça devrait être le même service. 

Par contre, si on doit envoyer une notification car l'utilisateur dépasse son quota de CO2, alors oui, ça n'a plus rien à voir avec le domaine de récupération des trips, et dans ce cas un autre service ferait sens. Mais un seul service "notif" pour gérer l'ensemble des notifs ne fait pas sens non plus. Le service === un domaine fonctionnel. Ça me parait pas mal comme approche